### DISCUSSION TITLE
Windows Start Menu

### DISCUSSION THREAD

**Discussion by Hottihotte** [valinet/ExplorerPatcher#4012] [General]
Start menu is set to "Start menu style*:Windows 10".
The Microsoft Store is pinned in the start menu.
But you can't "unpin it from start" because you can only "pin it to start" even though it is already pinned.
What can I do to get rid of it?
Windows Version Windows 11 23H2 Build 22631.4541
EP Version 22621.4317.67.1
![Screenshot 2024-12-08 030116](https://github.com/user-attachments/assets/c9d298bb-0c03-4b85-9358-1532bbf4052b)

**Comment by Hottihotte** [comment]
I ask for help.

**Comment by milestprower92** [comment]
https://github.com/valinet/ExplorerPatcher/discussions/1679

**Reply by Hottihotte** [reply]
> #1679

That's not the problem.

Recently, the Stor could still be removed from the start.

It was either before the Windows update or before the EP update.

https://github.com/valinet/ExplorerPatcher/discussions/1679#discussioncomment-7159448
didn't help

**Reply by milestprower92** [reply]
what i meant is that for 10 start menu-related issues use that discussion

**Comment by Hottihotte** [comment]
has been resolved, everything is back to normal
